src/me/corriekay/pokegoutil/windows/PokemonTab.java

```diff
@@ -850,10 +862,13 @@ private void constructNewTableModel(PokemonGo go, List<Pokemon> pokes) {
                 caughtCol = new ArrayList<>(),//22
                 favCol = new ArrayList<>();//23
         private final ArrayList<Long> duelAbilityCol = new ArrayList<>();//24
-        private final ArrayList<Long> gymOffenseCol = new ArrayList<>();//25
+        private final ArrayList<Double> gymOffenseCol = new ArrayList<>();//25
```

Comment:
Why does gymOffense has to be a Double? Thought it is a number without decomals too?